languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt

```diff
@@ -5236,3 +5236,159 @@ Legacys	Legacy	SUB:NOM:PLU:FEM
 Legacys	Legacy	SUB:GEN:PLU:FEM
 Legacys	Legacy	SUB:DAT:PLU:FEM
 Legacys	Legacy	SUB:AKK:PLU:FEM
+begründ	begründen	VER:IMP:SIN:SFT
+bewert	bewerten	VER:IMP:SIN:SFT
+gestalt	gestalten	VER:IMP:SIN:SFT
+erläuter	erläutern	VER:IMP:SIN:SFT
+glieder	gliedern	VER:IMP:SIN:SFT
+ermittel	ermitteln	VER:IMP:SIN:SFT
+arbeit	arbeiten	VER:IMP:SIN:SFT
+aufstöhnen	aufstöhnen	VER:INF:SFT
+aufgestöhnt	aufstöhnen	VER:PA2:SFT
+aufstöhnend	aufstöhnen	VER:PA1:SFT
+aufzustöhnen	aufstöhnen	VER:EIZ:SFT
+verlasse	verlassen	VER:IMP:SIN:NON
+lasse	lassen	VER:IMP:SIN:NON
+lasse	lassen	VER:MOD:IMP:NON
+verbesser	verbessern	VER:IMP:SIN:SFT
+heil	heilen	VER:IMP:SIN:SFT
+gelassenen	gelassen	ADJ:NOM:PLU:FEM:GRU:DEF
+gelassenen	gelassen	ADJ:NOM:PLU:MAS:GRU:DEF
+gelassenen	gelassen	ADJ:NOM:PLU:NEU:GRU:DEF
+gelassenen	gelassen	ADJ:GEN:PLU:FEM:GRU:DEF
+gelassenen	gelassen	ADJ:GEN:PLU:MAS:GRU:DEF
+gelassenen	gelassen	ADJ:GEN:PLU:NEU:GRU:DEF
+gelassenen	gelassen	ADJ:DAT:PLU:FEM:GRU:DEF
+gelassenen	gelassen	ADJ:DAT:PLU:MAS:GRU:DEF
+gelassenen	gelassen	ADJ:DAT:PLU:NEU:GRU:DEF
+gelassenen	gelassen	ADJ:AKK:PLU:FEM:GRU:DEF
+gelassenen	gelassen	ADJ:AKK:PLU:MAS:GRU:DEF
+gelassenen	gelassen	ADJ:AKK:PLU:NEU:GRU:DEF
+gelassenen	gelassen	ADJ:NOM:PLU:FEM:GRU:IND
+gelassenen	gelassen	ADJ:NOM:PLU:MAS:GRU:IND
+gelassenen	gelassen	ADJ:NOM:PLU:NEU:GRU:IND
+gelassenen	gelassen	ADJ:GEN:PLU:FEM:GRU:IND
+gelassenen	gelassen	ADJ:GEN:PLU:MAS:GRU:IND
+gelassenen	gelassen	ADJ:GEN:PLU:NEU:GRU:IND
+gelassenen	gelassen	ADJ:DAT:PLU:FEM:GRU:IND
+gelassenen	gelassen	ADJ:DAT:PLU:MAS:GRU:IND
+gelassenen	gelassen	ADJ:DAT:PLU:NEU:GRU:IND
+gelassenen	gelassen	ADJ:AKK:PLU:FEM:GRU:IND
+gelassenen	gelassen	ADJ:AKK:PLU:MAS:GRU:IND
+gelassenen	gelassen	ADJ:AKK:PLU:NEU:GRU:IND
+gelassene	gelassen	ADJ:NOM:PLU:FEM:GRU:SOL
+gelassene	gelassen	ADJ:NOM:PLU:MAS:GRU:SOl
```

Comment:
Hi @Dallun511, danke für diese schöne Sammlung!

Beim `:SOL` am Ende ist ab und zu das "L" klein geblieben, könntest du das bitte noch groß machen? Ist mir in der Liste ein paarmal ins Auge gefallen.

Wenn wir das Verb "aufbrezeln" dazunehmen, darunter die 1. Pers. Sg. "aufbrezele", sollten wir dann nicht auch "aufbrezle" für die Ich-Form haben?

Reply:
aufbrezle habe ich hinzugefügt, falls jemand diese Verform nutzt.
SOl wurde zu SOL.